lib/controller.js

```diff
@@ -176,6 +177,62 @@ may or may not be open and closes them
 
 */
 
+/*
+Reboots your tessel
+*/
+
+controller.reboot = function(opts) {
+  return new Promise(function(resolve, reject) {
+    Tessel.get(opts)
+      .then(function(tessel) {
+        var command = 'ssh root@' + tessel.lanConnection.ip + ' -i ' + opts.key + ' \'reboot ';
+        if (opts.time ||  opts.delay)  {
+          command = command + ' -d ' + opts.time || opts.delay;
+        }
+        if (opts.force) {
+          command = command + ' -f ';
+        }
+        if (opts.nosync) {
+          command = command + ' -n ';
```

Comment:
nit: `command = command + ' -n ';` => `command += ' -n ';` (and the others as well)